### PR TITLE
Fixed #11. Create and Update Vacancy Category 

### LINKED ISSUE #11: Basic CRUD for Vacancy Category

### PR DESCRIPTION
**Pull request by alisherrik**

### CODE REVIEW

File: Application/Features/VacancyCategories/Command/CreateVacancyCategory.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        _context = context;
+        _mapper = mapper;
+    }
+    public async Task<long> Handle(CreateVacancyCategoryCommand request, CancellationToken cancellationToken)
+    {
+        var vacancyCategory = _mapper.Map<VacancyCategory>(request);
+        var result = _context.Categories.Add(vacancyCategory);
+        await _context.SaveChangesAsync(cancellationToken);
+        return result.Entity.Id;
+    }
```
Comment (IslomMakhsudov) [inline Application/Features/VacancyCategories/Command/CreateVacancyCategory.cs:31]: update this code

File: WebUI/ClientApp/src/app/web-api-client.ts [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+            return blobToText(responseBlob).pipe(_observableMergeMap((_responseText: string) => {
+            return throwException("An unexpected server error occurred.", status, _responseText, _headers);
+            }));
+        }
+        return _observableOf(null as any);
+    }
+
+    update(id: number, request: UpdateJobVacancyCommand): Observable<number> {
+        let url_ = this.baseUrl + "/api/JobVacancies/{id}";
+        if (id === undefined || id === null)
```
Comment (IslomMakhsudov) [inline WebUI/ClientApp/src/app/web-api-client.ts:91]: test

**Review by IslomMakhsudov — COMMENTED** [review]
fix review notes

**Review by IslomMakhsudov — APPROVED** [review]

**Review by Muhammadislom99 — APPROVED** [review]

**Review by Muhammadislom99 — APPROVED** [review]
123

**Review by IslomMakhsudov — APPROVED** [review]

**Pull request merged**